### PR TITLE
Swap seekbar position with player buttons in Queue screen

### LINKED ISSUE #2931: [Feature Request/Android 10 Bug] Move the video controls at the very bottom of the screen up to prevent accidental touches by fullscreen users
Ever since the Android 10 update, whenever I'm playing a video in the background and have NewPipe open, the video controls such as the time along in the song & etc are at the very bottom of the screen. When I try and swipe up on my home screen (since I have a fullscreen phone), it will always accidently hit the bar and send my current song that I'm listening to into the middle of the song. It's getting to be a bit annoying, and I dunno why this only started with the latest Android 10 update.

Hopefully my description of the issue is accurate and understandable enough.

### LINKED ISSUE #6788: Full Screen Minimize Gesture Conflicts with Background Player Track Progress / Seek Bar
### Checklist

- [x] I am using the latest version - 0.21.7 
- [x] I have read the contribution guidelines given at https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md.
- [x] This issue contains only one bug. I will open one issue for every bug report I want to file.

### Steps to reproduce the bug
1. Try 'Start playing any video/music/playlist in the background'
2. Navigate and maximize/open 'Background player queue that is found in the notification bar'
3. If the phone has full screen gestures activated, and the selected video/music/playlist is running 'Try to minimize the currently open background player screen with a swiping action from bottom edge of the screen towards the middle'

### Actual behavior

The finger will interact with seekbar and make the track jump exactly to the position where the finger crossed it while attempting to minimize the background player screen/window.

### Expected behavior

Full screen gestures shall not overlap or interact with any of the navigational elements within the application. Otherwise it creates false inputs as stated above.

### Screenshots/Screen recordings

Screenshot_20210721-142840_NewPipe
[Video of the bug with 'Show visual feedback for taps' activated. (streamable.com)](https://streamable.com/hyk9e9) 

### Logs

N/A

### Device info
 - Android version: Android 11 / OneUI 3.1
 - Device model: Samsung Galaxy S10e

### PR DESCRIPTION
**Pull request by phigjm**
#### What is it?
- [x] Bugfix (user facing)
- [ ] Feature (user facing)
- [ ] Codebase improvement (dev facing)
- [ ] Meta improvement to the project (dev facing)

#### Description of the changes in your PR

- reorder the Controll elements of the Play queue (putting the Title down)

#### Before/After Screenshots/Screen Record

<table>
  <tr>
    <th>Before</th>
    <th>After</th>
  </tr>
  <tr>
    <td></td>
    <td></td>
  </tr>
</table>

#### Fixes the following issue(s)

- Fixes #6788 
- Fixes #2931
- Better view of the current playtime when seeking.

#### APK testing 

The APK can be found by going to the "Checks" tab below the title. On the left pane, click on "CI", scroll down to "artifacts" and click "app" to download the zip file which contains the debug APK of this PR.

#### Due diligence
- [x] I read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by blindm0use** [comment]
Great job fixing my issue!
It is best to place timecode popup dialog while seeking either in the middle of the screen or slightly above the seek bar. Otherwise vision gets obstructed by the finger while seeking. Thanks!

**Comment by phigjm** [comment]
> Great job fixing my issue!
> It is best to place timecode popup dialog while seeking either in the middle of the screen or slightly above the seek bar. Otherwise vision gets obstructed by the finger while seeking. Thanks!

I'll give it a try.

**Comment by phigjm** [comment]
Here you are
seek_Time

**Comment by SameenAhnaf** [comment]
@phigjm Thanks for opening a PR. I don't have a device with the bug. So, rectify me if I am wrong.

But I think, seek bar should be placed in the middle and preview should be shown within the title bar as it used to do. Google Podcasts works this way. Could you please give it a go?

IMG_20210803_183737

**Comment by Stypox** [comment]
@SameenAhnaf I agree with you, we should follow that ordering. Also, @phigjm try to make it look like as if there is the same spacing between seekbar-title and seekbar-buttons.

**Comment by phigjm** [comment]
This is how it looks now:
final

at the bottom, there is just enough space to fully represent the play button effects. These buttons seem to only trigger at release. So there should be no problem with the gesture navigation.
I also decided (as you can see) to move the Time above the title as it is transparent and therefore doesn't look that grate over the title.

**Comment by phigjm** [comment]
And here a screenshot from a simulated pixel a3:
Screenshot_1628115209

File: app/src/main/res/layout/activity_player_queue_control.xml [hunk 1]
```diff
@@ -31,17 +31,20 @@
...
-        android:layout_above="@id/center"
+        android:layout_above="@id/progress_bar"
         android:layout_below="@id/appbar"
         android:scrollbars="vertical"
         app:layoutManager="LinearLayoutManager"
         tools:listitem="@layout/play_queue_item" />
 
+
+
     <RelativeLayout
```
Comment (Stypox) [inline app/src/main/res/layout/activity_player_queue_control.xml:42]: Please reorder the `RelativeLayout`s in the code according to their visual order

**Review by Stypox — CHANGES_REQUESTED** [review]

**Comment by Stypox** [comment]
The last two screenshots look really good to me. Thank you! I've edited all of your comments to use `` for screenshots so that they aren't so big.

**Comment by phigjm** [comment]
> The last two screenshots look really good to me. Thank you! I've edited all of your comments to use `` for screenshots so that they aren't so big.

Thank you

File: app/src/main/res/layout/activity_player_queue_control.xml (hunk 1 above)
Comment (phigjm) [reply app/src/main/res/layout/activity_player_queue_control.xml:42]: > Please reorder the `RelativeLayout`s in the code according to their visual order

Sure, I check this later.

**Review by Stypox — APPROVED** [review]
I rebased and squashed; I un-nested some views (unrelated to your work). Thank you!

**Comment by litetex** [comment]
@Stypox 
Is this ready for merging?

**Comment by opusforlife2** [comment]
@litetex If he's approved it and not asked anyone else to review, then it must mean he considers it ready. Is that right, @Stypox?

**Pull request merged**